issuesync.rb

```diff
@@ -80,9 +80,11 @@ def format(io, issue, comments)
     end
 
     def format_body(io, issue)
-      title = "##{issue.number}: #{issue.title.strip}"
-      title << issue.labels
-      title << "  [CLOSED]" if issue.closed?
+      title = "##{issue.number}: #{issue.title.strip} "
+      for label in issue.labels
+        title << " [" + label + "]"
```

Comment:
Thanks, I'll update (after figuring out the spacing issue). My language of choice doesn't have `#{}` syntax, so its just not something I think of :smile: